modules/vpc/main.tf

```
@@ -13,6 +13,7 @@ module "vpc" {
 
   name = var.name_prefix
 
```

suggest that we add this flag as it is currently implicit would rather have it in the tf showing we need it.
```suggestion
enable_dns_support = true
```